pkg/ceph/client/status_test.go

```diff
@@ -24,7 +24,7 @@ import (
 
 const (
 	// this JSON was generated from the mon_command "status",  ExecuteMonCommand(conn, map[string]interface{}{"prefix": "status"})
```

Comment:
this comment can probably be updated for how this output was generated (i bet it was `ceph status --format=json`)